charts/trino/Chart.yaml

```diff
@@ -15,9 +15,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.2.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 351
+appVersion: "355"
```

Comment:
Nit: are quotes needed?

Reply:
not exactly, but helm lint advise so:
`[ERROR] Chart.yaml: appVersion should be of type string but it's of type float64`